lib/solvers/NetLabelPlacementSolver/SingleNetLabelPlacementSolver/geometry.ts

```diff
@@ -1,18 +1,44 @@
 import type { FacingDirection } from "lib/utils/dir"
+import { estimateTextWidth } from "lib/utils/estimate-text-width"
+import { getSchMmFontSize } from "lib/utils/get-sch-font-size"
 
-export const NET_LABEL_HORIZONTAL_WIDTH = 0.45
 export const NET_LABEL_HORIZONTAL_HEIGHT = 0.2
+// Constants matching the net label implementation
+export const ARROW_POINT_WIDTH_FSR = 0.1
+export const END_PADDING_FSR = 0.05
+export const END_PADDING_EXTRA_PER_CHARACTER_FSR = 0.01
```

Comment:
yea just keep all of this in core, core already does this and it belongs in core